nextflow_schema.json

```diff
@@ -104,6 +104,19 @@
             },
             "fa_icon": "far fa-clipboard"
         },
+        "fastmatch": {
+            "title": "FastMatch",
+            "type": "object",
+            "description": "Parameters for FastMatch",
+            "default": "",
+            "properties": {
+                "threshold": {
+                    "type": "number",
```

Comment:
Could you set a `"minimum"` threshold of 0 here: https://nextflow-io.github.io/nf-validation/nextflow_schema/nextflow_schema_specification/#minimum-maximum